valkey/valkey.conf

```diff
@@ -0,0 +1,4 @@
+cluster-enabled yes
+cluster-config-file nodes.conf
```

Comment:
What is nodes.conf? Should it be in repo?

Reply:
Or maybe we can inline 1 line into compose config?

Reply:
Here's a way to make it all work

Reply:
Actually valkey.conf provides minimal configuration to make cluster up and running